ee/clickhouse/queries/event_query.py

```diff
@@ -41,6 +46,8 @@ def __init__(
 
         self._should_join_distinct_ids = should_join_distinct_ids
         self._should_join_persons = should_join_persons
+        self._extra_fields = extra_fields
+        self._extra_person_fields = extra_person_fields
```

Comment:
I see that `self._extra_person_fields` is being used within the class but I don't see `self._extra_fields` being used, do we need it?

Reply:
Also, what does extra_person_fields mean? Is it just that these fields will be selected? Perhaps a comment to say so would be useful

Reply:
`_extra_fields` is being used in a subclass when constructing the fields to select from (events) table :)

Will add a comment